test/test_source.vim

```diff
@@ -0,0 +1,31 @@
+let s:vimlparser = vimlparser#import()
+
+function! s:run()
+  let src = [
+  \  'function! s:load_on()',
+  \  '  imap <expr> <Tab> neosnippet#expandable_or_jumpable() ?',
+  \  '\ "\<Plug>(neosnippet_expand_or_jump)" : "\<Tab>"',
+  \  'endfunction',
```

Comment:
Can you add different test cases which contains spaces before `\`. 
Or simply add space before `\` in this test case

Reply:
okay. done